examples/nlp/language_modeling/conf/megatron_model_base_config.yaml

```diff
@@ -36,4 +36,5 @@ megatron_legacy: False # Whether to use the legacy Megatron model. This affects
 normalize_attention_scores: True # Whether to scale the output Q * K^T by 1 / sqrt(hidden_size_per_head). This arg is provided as a configuration option mostly for compatibility with models that have been weight-converted from HF. You almost always want to se this to True.
 num_moe_experts: 1 # When >1, FFNs are changed to MoE layers
 moe_frequency: 1 # every Nth ffn layer will be made MoE 
-moe_dropout: 0.0 # Dropout value for MoE layers
\ No newline at end of file
+moe_dropout: 0.0 # Dropout value for MoE layers
+use_flash_attention: false # Use flash attention in self-attention module
```

Comment:
```suggestion
use_flash_attention: false # Use flash attention in self-attention module, this config does nothing when transformer_engine=True
```

Reply:
I add this comment in `examples/nlp/language_modeling/conf/megatron_gpt_config.yaml` since `transformer_engine` only used for GPT models